src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -226,7 +246,7 @@ private TimestampInterval createTimestampInterval ( cstring str_begin,
 
     if ( !isRelativeTime(str_end) )
     {
-        reference_time = parseDateString(str_end, include_end_date);
+        reference_time = parseDateString(removeTimeDivision(str_end), include_end_date);
```

Comment:
Same question as above, don't we remove division only from "relative time" values?

Reply:
it's needed because `parseDateString` does not know how to parse strings with time divisions